src/server/JasmineGraphInstanceService.cpp

```diff
@@ -939,7 +840,7 @@ int JasmineGraphInstanceService::collectTrainedModelThreadFunction(instanceservi
 
 void JasmineGraphInstanceService::createPartitionFiles(std::string graphID, std::string partitionID,
                                                        std::string fileType) {
-    instance_logger.log(fileType, "info");
+    instance_logger.info("createPartitionFiles called with fileType=" + fileType);
```

Comment:
This info log message does not have any value. Lets remove it.

Reply:
Removed it

Reply:
Ok